source/puppet/5.3/release_notes.markdown

```diff
@@ -20,6 +20,51 @@ Read the [Puppet 5.1](/puppet/5.1/release_notes.html) and [Puppet 5.2 release no
 
 Also of interest: the [Puppet 4.10 release notes](/puppet/4.10/release_notes.html) and [Puppet 4.9 release notes](/puppet/4.9/release_notes.html).
 
+## Puppet 5.3.3
+
+Released November 6, 2017.
+
+This is a bug-fix release of Puppet.
+
+-   [All issues resolved in Puppet 5.3.3](https://tickets.puppetlabs.com/issues/?jql=fixVersion%20%3D%20%27PUP%205.3.3%27)
+
+### Bug fix: Puppet can install Forge modules with many available versions
```

Comment:
Starting with 5.0, I started to try and align open source release notes with PE's release notes style guide, so my suggestions are based on that, as well as ways to make these easier to write and maintain in easyDITA later on. I realize there's not an official confluence page for open source release note guide like there is for PE, I was mostly just closely mimicking that so I didn't think it warranted its own page, but if you want, I can totally create one. 

* There are 5ish h3 headings for open source: New features, Enhancements, Known issues (Which are now on their own page), Deprecations, and Bug fixes. I'd group all of these tickets you're listing under one 'Bug fixes' h3 so you're not repeating yourself with a "Bug fix:" in every heading and bump these down to h4s.
  * Additionally, you can make the short description of the Bug fixes topic something like "These are the issues resolved in this release." Then, you don't have to append repetitive statements like "This is fixed" in individual notes, it can be assumed based on the short desc.
* We also made the decision to stop including links to every individual ticket, especially the lower priority ones where we really are describing the whole gist of the issue in the release note. Since the link to all issues resolved in Product.x is available to them, readers can still get the info they want from that one link.

Reply:
@jtappa It would be awesome if our open source release notes conventions were standardized and documented somewhere public, even if they're the same as PE's.